### ISSUE TITLE
wrong screen color

### ISSUE THREAD

**Issue by monfiston** [OpenEVSE/openevse_esp32_firmware#943]
hello
i have 2 openEVSE, one official and one tinkered by myself
the color of the screen is not identical on the 2 openevse,
is there a way to change the color of the screen?

![image](https://github.com/user-attachments/assets/d8dd274b-ca8c-487f-a40c-8ffd120c3f8b)

**Comment by jeremypoulter** [comment]
Is the LCD an OpenEVSE LCD on both?

**Comment by monfiston** [comment]
No, it's not an original screen, but I thought that in the past you could choose a specific color?

**Comment by jeremypoulter** [comment]
The LCD colour is linked to the EVSE state. As it is lighting up and displaying text it must be the same driver chip, but I assume the RGB backlight is connected to the driver in a different way (I think the red and green are swapped). So for proper operation you will need to do a custom version of the OpenEVSE that corrects for this.

**Comment by monfiston** [comment]
hello
it is not possible to change the connection to the rgb led?
if not, what should I change in the code?
